tests/test_validators.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+from log_outgoing_requests.validators import (
+    _validate_content_length,
+    _validate_content_type,
+)
```

Comment:
I don't see the point in unit testing private helper functions, all you are doing is tying your tests to your implementation details.

Shouldn't you instead just call the public function and set up your test accordingly so that either the content-length/content-type part passes and the other fails? Ultimately that's the outcome you expect and how you get there is irrelevant for the perspective of the test.

I would even go further and parametrize my tests extensively:

```py
@pytest.mark.parametrize(
    "max_cl,allowed_ct,content_length,content_type,expected",
    [
        (2048, ["application/json"], "1000", "APPLICATION/JSON", True),
        ...  # other variants
    ],
)
def test_content_type_content_length_check(max_cl, allowed_ct, content_length, content_type, expected):
    config = OutgoingRequestsLogConfig(max_content_length=max_cl, allowed_content_types=allowed_ct)

    result = validate_content(content_length, content_type, config)

    assert result == expected
```